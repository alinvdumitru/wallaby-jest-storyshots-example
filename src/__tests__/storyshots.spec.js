/* eslint-env jest */
import initStoryshots from 'storyshots';

initStoryshots();
