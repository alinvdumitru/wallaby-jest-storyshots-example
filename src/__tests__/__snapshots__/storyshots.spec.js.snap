// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots TestComponent TestComponent render 1`] = `
<div
  className="root"
>
  <div
    className="string"
  >
    String is 
  </div>
  <div
    className="number"
  >
    Number is 
  </div>
</div>
`;
